### ISSUE TITLE
[YSQL]  Support:  CREATE LANGUAGE not supported yet

### ISSUE THREAD

**Issue by rongfengliang** [yugabyte/yugabyte-db#2900]
Jira Link: [DB-1743](https://yugabyte.atlassian.net/browse/DB-1743)
i want test plv8 extension with ysql

```code
CREATE EXTENSION plv8;
``` 
got 
```code
ERROR:  CREATE LANGUAGE not supported yet
HINT:  Please report the issue on https://github.com/YugaByte/yugabyte-db/issues
```

have any plan for support `CREATE LANGUAGE` ? 

Thanks !

**Comment by shaunstoltz** [comment]
This is resolved by this pr https://github.com/yugabyte/yugabyte-db/pull/9576

**Comment by radekg** [comment]
> This is resolved by this pr #9576

This one isn't. My PR does not add support for `CREATE LANGUAGE`, it merely ensures that an extension is properly loaded via the `-c shared_preload_libraries` in PgWrapper.

**Comment by shaunstoltz** [comment]
Sure, but I think that "blah language no supported yet" is default message when any none hard coded extension is attempted to be "created". So, I think that the create support is there, it is just failing with this default language because of the inability to handle user passed libraries, which your PR address as far as I can tell?

Anyway, hoping that is the case as I also have the same issue with other extensions I try. And thanks for this contribution... awesome!

**Comment by radekg** [comment]
> Sure, but I think that "blah language no supported yet" is default message when any none hard coded extension is attempted to be "created". So, I think that the create support is there, it is just failing with this default language because of the inability to handle user passed libraries, which your PR address as far as I can tell?

I could be wrong but I do not think this is the case. There is no explicit `create language` statement in any plv8 SQL code but you can see that the `create language` statement is unsupported on the YugabyteDB grammar level: https://github.com/yugabyte/yugabyte-db/blob/master/src/postgres/src/backend/parser/gram.y#L4712.

**Comment by rongfengliang** [comment]
> > Sure, but I think that "blah language no supported yet" is default message when any none hard coded extension is attempted to be "created". So, I think that the create support is there, it is just failing with this default language because of the inability to handle user passed libraries, which your PR address as far as I can tell?
> 
> I could be wrong but I do not think this is the case. There is no explicit `create language` statement in any plv8 SQL code but you can see that the `create language` statement is unsupported on the YugabyteDB grammar level: https://github.com/yugabyte/yugabyte-db/blob/master/src/postgres/src/backend/parser/gram.y#L4712.
> 

create language is a pg buildin feature

**Comment by radekg** [comment]
> create language is a pg buildin feature

Yes, it is a built-in feature. Not explicitly invoked by the plv8 sql scripts, maybe via C++ API and disabled in YugabyteDB at the grammar level.

**Comment by discountscott** [comment]
We are also trying to add plv8 as it's required for one of our applications.

Any potential to add support for ```create language``` so that it can be added?

**Comment by kriteshranjan** [comment]
Closing this issue based on AI validation. cc: @m-iancu , @VamsidharGunturu
